app/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
         applicationId "com.cobresun.factfulnewsandroid"
         minSdkVersion 23
         targetSdkVersion 28
-        versionCode 4
+        versionCode 5
         versionName "1.0"
```

Comment:
I just realized we forgot to update `versionName` in this one, not that it's a big deal, but we should remember for future, since its whats displayed on the store page!